### PR TITLE
Add org.kde.CrowTranslate

### PR DESCRIPTION
**Pull request by Shatur**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
A simple and lightweight translator that allows you to translate and speak text.
- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project.

The application was previously submitted under [io.crow_translate.CrowTranslate](https://github.com/flathub/io.crow_translate.CrowTranslate), but I need to change the appid due to the transition under the KDE umbrella. After creating a new repository, I will send a PR with a eol rebase manifest to the previous repository. So users will be notified about the appid change.

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.crow_translate.CrowTranslate.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/crow-translate/crow-translate.git
+        tag: 2.11.1
+        commit: 7113ff01a994584ed0a3ec557fc3a694fcb06b63
+        x-checker-data:
+          type: git
```
Comment (bbhtt) [inline io.crow_translate.CrowTranslate.yaml:36]: This repo is archived, I'm guessing this should be using KDE provided tarballs.

**Comment by bbhtt** [comment]
cc @flathub/kde

File: io.crow_translate.CrowTranslate.yaml (hunk 1 above)
Comment (bbhtt) [reply io.crow_translate.CrowTranslate.yaml:36]: Also is this the source now https://invent.kde.org/office/crow-translate ? (That'd justify the org.kde. ID)

Comment (Shatur) [reply io.crow_translate.CrowTranslate.yaml:36]: > This repo is archived, I'm guessing this should be using KDE provided tarballs.

It's because we don't have a new release yet. But I want to create a flathub entry first, otherwise [the app page](https://apps.kde.org/crowtranslate/) will point to an old release.

> 

> Also is this the source now https://invent.kde.org/office/crow-translate ? (That'd justify the org.kde. ID)

Yes!

**Comment by hfiguiere** [comment]
cc @flathub/kde

File: io.crow_translate.CrowTranslate.yaml (hunk 1 above)
Comment (travier) [reply io.crow_translate.CrowTranslate.yaml:36]: Then please use invent as the source, with a fixed commit, not GitHub

File: io.crow_translate.CrowTranslate.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS:BOOL=ON
+        cleanup:
+          - /bin
+        sources:
+          - type: git
+            url: https://github.com/tesseract-ocr/tesseract.git
+            tag: 5.3.4
+            commit: 8ee020e14cf5be4e3f0e9beb09b6b050a1871854
```
Comment (travier) [inline io.crow_translate.CrowTranslate.yaml:49]: Please use release tarballs as much as possible, not direct Git repos.

File: io.crow_translate.CrowTranslate.yaml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+copy-icon: true
+command: crow
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
```
Comment (travier) [inline io.crow_translate.CrowTranslate.yaml:28]: This should come last

File: io.crow_translate.CrowTranslate.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+          - cp -av * ${FLATPAK_DEST}/share/tessdata/
+        sources:
+          - type: git
+            url: https://github.com/tesseract-ocr/tessdata_fast.git
+            tag: 4.1.0
+            commit: 65727574dfcd264acbb0c3e07860e4e9e9b22185
+            x-checker-data:
+              type: git
+
+      - name: extra-cmake-modules
```
Comment (travier) [inline io.crow_translate.CrowTranslate.yaml:84]: This is already in the SDK

File: io.crow_translate.CrowTranslate.yaml (hunk 3 above)
Comment (Shatur) [reply io.crow_translate.CrowTranslate.yaml:28]: Could you elaborate?

File: io.crow_translate.CrowTranslate.yaml (hunk 1 above)
Comment (Shatur) [reply io.crow_translate.CrowTranslate.yaml]: Switched to using invent. I realized that we imported previous releases, so I can just change the URL and it will work.

File: io.crow_translate.CrowTranslate.yaml (hunk 4 above)
Comment (Shatur) [reply io.crow_translate.CrowTranslate.yaml]: Removed!

File: io.crow_translate.CrowTranslate.yaml (hunk 2 above)
Comment (Shatur) [reply io.crow_translate.CrowTranslate.yaml]: Switched to tarballs. I just used previously submitted yaml.

File: io.crow_translate.CrowTranslate.yaml (hunk 3 above)
Comment (bbhtt) [reply io.crow_translate.CrowTranslate.yaml:28]: > This should come last

This is the only toplevel module. It's using nested modules.

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

File: io.crow_translate.CrowTranslate.yaml
Comment (bbhtt) [inline io.crow_translate.CrowTranslate.yaml:1]: Manifest filename is wrong `org.kde.CrowTranslate.yaml`

**Comment by bbhtt** [comment]
Something is wrong with build requests, I'll start it manually once the filename is fixed.

**Comment by bbhtt** [comment]
Ah Github is having an outage https://www.githubstatus.com/

File: io.crow_translate.CrowTranslate.yaml
Comment (Shatur) [reply io.crow_translate.CrowTranslate.yaml:1]: Updated, thanks!

**Comment by Shatur** [comment]
Sorry, I accidentally calculated SHA1 instead. Fixed.

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

File: org.kde.CrowTranslate.yaml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://invent.kde.org/office/crow-translate/.git
```
Comment (bbhtt) [inline org.kde.CrowTranslate.yaml:32]:
```suggestion
        url: https://invent.kde.org/office/crow-translate.git
```

File: org.kde.CrowTranslate.yaml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+            builddir: true
+            config-opts:
+              - -DBUILD_SHARED_LIBS:BOOL=ON
+            sources:
+              - type: archive
+                url: https://github.com/DanBloomberg/leptonica/archive/refs/tags/1.84.1.tar.gz
+                sha256: ecd7a868403b3963c4e33623595d77f2c87667e2cfdd9b370f87729192061bef
+
+      - name: tessdata_fast
+        disabled: true
```
Comment (bbhtt) [inline org.kde.CrowTranslate.yaml:64]: If you are skipping this why is this added here?

Comment (Shatur) [reply org.kde.CrowTranslate.yaml:64]: I just used what was added here: https://github.com/flathub/io.crow_translate.CrowTranslate/blob/ce7138a8686f7858ad2f99ce03ee11f5b8413f88/io.crow_translate.CrowTranslate.yaml#L70
Didn't even notice that it's disabled. Should I just remove it?

Comment (bbhtt) [reply org.kde.CrowTranslate.yaml:64]: I have no idea if you need it or not. Why was it added to the manifest and why was it disabled?

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

**Comment by Shatur** [comment]
Hm... Looks like it's unable to find the .desktop file.
Why it could be?
I generate it [here](https://invent.kde.org/office/crow-translate/-/blob/master/CMakeLists.txt?ref_type=heads#L253).
And install it [here](https://invent.kde.org/office/crow-translate/-/blob/master/CMakeLists.txt?ref_type=heads#L256).

File: org.kde.CrowTranslate.yaml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
+rename-icon: crow-translate
```
Comment (bbhtt) [inline org.kde.CrowTranslate.yaml:18]:
```suggestion
  #- --talk-name=org.gnome.Shell
rename-desktop-file: io.crow_translate.CrowTranslate.desktop
rename-icon: crow-translate
```

Comment (Shatur) [reply org.kde.CrowTranslate.yaml:18]: But why? Isn't appid is `org.kde.CrowTranslate`?

Comment (bbhtt) [reply org.kde.CrowTranslate.yaml:18]: the desktop file you are installing still has the old name...

Did you build this locally or just checked the 4th checkbox?

Comment (Shatur) [reply org.kde.CrowTranslate.yaml:18]: > the desktop file you are installing still has the old name...

Ah, it's because I using the old commit that corresponds to the version with the old appid. What will be the best way to proceed? Add `rename-desktop-file` for now and remove after the release? Or should I pin to the latest commit?

> Did you build this locally or just checked the 4th checkbox?

I built it on KDE CI, but didn't check after making all these changes, sorry.

Comment (bbhtt) [reply org.kde.CrowTranslate.yaml:18]: > Add rename-desktop-file for now and remove after the release? 

Yea that's fine. We can proceed with the build and test. You can remove it later.

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

**Comment by bbhtt** [comment]
I'm guessing the metainfo is still using the old name too, use `rename-appdata-file` similarly in that case.

**Comment by Shatur** [comment]
Makes sense, on this commit everything corresponds to the old ID.

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

**Comment by Shatur** [comment]
Looks like something changed since the last submission and I need to add more fields. Let me try to run the linter locally.

**Comment by Shatur** [comment]
@bbhtt I can update my appdata, but it's located in the repository. Should I try to use the latest commit instead?
Or I can temporarily add it here.

**Comment by bbhtt** [comment]
You can add it as a patch https://docs.flatpak.org/en/latest/module-sources.html#patch-sources

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

**Comment by Shatur** [comment]
> Tray is missing and icon and tray menu is missing icon on GNOME 46/Fedora 40 VM. I guess this would work in a KDE system.

Yes, I just tested the package on KDE and it works correctly. But it's bad that it doesn't work on GNOME properly. I remember [this](https://github.com/crow-translate/crow-translate/issues/665) issue and installing from PPA helped. Do I need to adjust something in the flatpak configuration to make it work with GNOME?

> Also why is it opening a tray by default? Why not show the window first by default?

It's a good question! I personally keep it enabled because I run the application automatically when the system starts. But the autostart itself is disabled by default. So your suggestion makes total sense. Just disabled "Start minimized" by default.

File: io.crow_translate.CrowTranslate.yaml (hunk 3 above)
Comment (travier) [reply io.crow_translate.CrowTranslate.yaml]: Ah, good point, I hadn't realized that. Good then

**Comment by travier** [comment]
I don't think it's worth spending time patching the old version. Let's get all the fixes upstream and use the latest commit here even if it's not released yet and we'll move to a release tarball once it's out.

File: org.kde.CrowTranslate.yaml [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
```
Comment (travier) [inline org.kde.CrowTranslate.yaml:13]: Let's sort this list

File: org.kde.CrowTranslate.yaml [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
```
Comment (travier) [inline org.kde.CrowTranslate.yaml:17]: This should go into a README for the repo to make it easier to find.

Maybe also in a message in the application when running as Flatpak.

File: org.kde.CrowTranslate.yaml [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+          type: git
+    modules:
+      - name: tesseract
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS:BOOL=ON
+        cleanup:
+          - /bin
+        sources:
```
Comment (travier) [inline org.kde.CrowTranslate.yaml:48]: You should add `x-checker-data` config for all sources

**Comment by bbhtt** [comment]
> Do I need to adjust something in the flatpak configuration to make it work with GNOME?

Not it should work with the current configuration? What icon name are using for the tray? The icon needs to be exported by Flatpak on host so that it is available in XDG_DATA_DIRS. The tray extension cannot access an icon that Flatpak hasn't exported.

Flatpak will export icons from `$prefix/share/icons/hicolor/...` with the following patterns `$FLATPAK_ID, $FLATPAK_ID.foo, $FLATPAK_ID-foo` .png/.svg

**Comment by Shatur** [comment]
Ah, I use `crow-translate` as the icon. That's the issue.
Will change it switch to the latest master.

File: org.kde.CrowTranslate.yaml (hunk 9 above)
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:17]: Removed completely. `org.kde.KWin` and `org.gnome.Shell` aren't necessary.

File: org.kde.CrowTranslate.yaml (hunk 10 above)
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:48]: I checked other projects and looks like they don't use `x-checker-data` for archives from GitHub.
For example, [org.flathub.flatpak-external-data-checker](https://github.com/flathub/org.flathub.flatpak-external-data-checker/blob/502b06d4d4e67372ee1c699be27d920995c70e90/org.flathub.flatpak-external-data-checker.yml#L159C27-L159C34) does not do this.

**Comment by Shatur** [comment]
Done! Thank you a lot for the feedback. I'm new to Flatpak and looks like the app wasn't properly packaged before.
But it should be good now. I switched to the master and fixed the mentioned issues.

File: org.kde.CrowTranslate.yaml [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+copy-icon: true
+command: crow
+separate-locales: false
```
Comment (bbhtt) [inline org.kde.CrowTranslate.yaml:24]: Why is this disabled?

File: org.kde.CrowTranslate.yaml [hunk 12]
```diff
@@ -0,0 +1,87 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+add-extensions:
+  org.kde.CrowTranslate.tessdata:
```
Comment (bbhtt) [inline org.kde.CrowTranslate.yaml:6]: What is this for?

File: org.kde.CrowTranslate.yaml (hunk 11 above)
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:24]: The is unable to load localization without this line.

File: org.kde.CrowTranslate.yaml (hunk 12 above)
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:6]: It's needed for tesseract models for screen reading. I used [this](https://github.com/flathub/io.github.manisandro.gImageReader/blob/master/io.github.manisandro.gImageReader.yml) app, which also uses Tesseract, as a reference.

File: org.kde.CrowTranslate.yaml (hunk 11 above)
Comment (bbhtt) [reply org.kde.CrowTranslate.yaml:24]: Are you sure? You need to install the `.Locale` extension for that locale. By default Flatpak will only install the locale extension for your host locale.

This disabled creating locale extension and bundles all locales into the app.

Comment (Shatur) [reply org.kde.CrowTranslate.yaml:24]: Ah, I didn't know about it! In my application I have locale selection and it could be confusing when user picks a localization and it doesn't work.
Should I keep `separate-locales: false` or remove it?

Comment (bbhtt) [reply org.kde.CrowTranslate.yaml:24]: Then keep it, usually apps follow the system locale.

**Comment by Shatur** [comment]
Anything else needed from me?
Should we wait for the release drop before creating a repository?

File: org.kde.CrowTranslate.yaml [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+            sources:
+              - type: archive
+                url: https://github.com/DanBloomberg/leptonica/releases/download/1.84.1/leptonica-1.84.1.tar.gz
+                sha256: 2b3e1254b1cca381e77c819b59ca99774ff43530209b9aeb511e1d46588a64f6
+                x-checker-data:
+                  type: anitya
+                  project-id: 1549
+                  url-template: https://github.com/DanBloomberg/leptonica/releases/download/$version/leptonica-$version.tar.gz
+
+      # The `tessdata_fast` variant is recommended for end users: https://tesseract-ocr.github.io/tessdoc/Data-Files
```
Comment (travier) [inline org.kde.CrowTranslate.yaml:72]: Can you move that to a README file?

File: org.kde.CrowTranslate.yaml [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+      - name: tessdata_fast
+        buildsystem: simple
+        build-commands:
+          - rm -rf ${FLATPAK_DEST}/share/tessdata
+          - mkdir -p ${FLATPAK_DEST}/share/tessdata
+          - cp -av * ${FLATPAK_DEST}/share/tessdata/
+        sources:
+          - type: git
+            url: https://github.com/tesseract-ocr/tessdata_fast
```
Comment (travier) [inline org.kde.CrowTranslate.yaml:81]: Can this be fetched from a tarball instead of Git?

File: org.kde.CrowTranslate.yaml [hunk 15]
```diff
@@ -0,0 +1,87 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+add-extensions:
+  org.kde.CrowTranslate.tessdata:
+    directory: share/tessdata
+    version: stable
+    versions: stable;beta;test
```
[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.kde.CrowTranslate.yaml:9]: but then there is no such package. not even in the submission queue.

**Comment by travier** [comment]
> Anything else needed from me? Should we wait for the release drop before creating a repository?

We can "merge" this once this is ready, which is once the @flathub/kde team is giving it 👍🏻 and then we'll ask Flathub Admins to trigger the repo creation.

File: org.kde.CrowTranslate.yaml (hunk 15 above)
[the remark this answers is not in the document]
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:9]: Yes, I copied it from [this](https://github.com/flathub/io.github.manisandro.gImageReader/blob/master/io.github.manisandro.gImageReader.yml) app.

> but then there is no such package. not even in the submission queue.

Ah, I thought that it creates a extension for the app. Do I need to package tessdata separately? I don't see ` io.github.manisandro.gImageReader.tessdata` anywhere as well...

I'm new to flatpak.

[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.kde.CrowTranslate.yaml:9]: two wrongs don't make a right.

File: org.kde.CrowTranslate.yaml (hunk 13 above)
Comment (travier) [reply org.kde.CrowTranslate.yaml:72]: Ah, this copied from https://github.com/flathub/io.github.manisandro.gImageReader/blob/master/io.github.manisandro.gImageReader.yml?rgh-link-date=2024-07-09T12%3A52%3A57Z but I don't know how this works.

File: org.kde.CrowTranslate.yaml (hunk 15 above)
[the remark this answers is not in the document]
Comment (travier) [reply org.kde.CrowTranslate.yaml:9]: Let's try to figure out how to do properly. I'm not familiar with this so we'll have to search in the docs.

[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.kde.CrowTranslate.yaml:9]: `man flatpak-metadata`

[...]

```
       version (string)
           The branch to use when looking for the extension. If this is not specified, it defaults to the branch of the application
           or runtime that the extension point is for. Available since 0.4.1.

       versions (string)
           The branches to use when looking for the extension. If this is not specified, it defaults to the branch of the
           application or runtime that the extension point is for. Available since 0.9.1, and backported to the 0.8.x branch in
           0.8.4.

```

[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.kde.CrowTranslate.yaml:9]: What it doesn't say is that they are equivalent one being the plural form.

That doesn't make an extension being built. That just declare the extension point.

`man flatpak-manifest`

```
       bundle (boolean)
           If this is true, then the data created in the extension directory is omitted from the result, and instead packaged in a
           separate extension.

```

[the remark this answers is not in the document]
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:9]: > Let's try to figure out how to do properly

Reading it right now. Looks like I need to create a separate manifest for it and add a special entry to my app metadata.
We probably need to notify the maintainer of `io.github.manisandro.gImageReader` as well about it.

> man flatpak-metadata

We aren't talking about the version string, but about the whole `add-extensions` entry.

[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.kde.CrowTranslate.yaml:9]: > Reading it right now. Looks like I need to create a separate manifest for it and add a special entry to my app metadata.

no.

[the remark this answers is not in the document]
Comment (Shatur) [reply org.kde.CrowTranslate.yaml:9]: > That doesn't make an extension being built. That just declare the extension point.

It's because `bundle` is false by default?
So instead of setting a version and creating a separate manifest, I can replace `version` with `bundle: true`?

File: org.kde.CrowTranslate.yaml (hunk 14 above)
Comment (Shatur) [reply org.kde.CrowTranslate.yaml]: Fixed, thanks!

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

File: org.kde.CrowTranslate.yaml (hunk 15 above)
[the remark this answers is not in the document]
Comment (bbhtt) [reply org.kde.CrowTranslate.yaml]: > Yes, I copied it from [this](https://github.com/flathub/io.github.manisandro.gImageReader/blob/master/io.github.manisandro.gImageReader.yml) app.

See the tip here https://docs.flatpak.org/en/latest/extension.html#extension-point Since you don't have a beta application you don't need this. `test` is not needed, it's a temporary repo.

> It's because bundle is false by default?

Yes

https://docs.flatpak.org/en/latest/extension.html#bundled-extensions

[the remark this answers is not in the document]
Comment (Shatur) [reply org.kde.CrowTranslate.yaml]: I think I figured it out. I enabled bundling and now the extension creates properly. I also set version that matches the tessdata version.

Opened an issue no notify the maintainer of io.github.manisandro.gImageReader about it: https://github.com/flathub/io.github.manisandro.gImageReader/issues/70

@hfiguiere Thanks for pointing it out, but if your goal is to help the community, try to give helpful advice to newcomers instead of just saying "it's wrong".

File: org.kde.CrowTranslate.yaml (hunk 13 above)
Comment (Shatur) [reply org.kde.CrowTranslate.yaml]: Done!

File: org.kde.CrowTranslate.yaml (hunk 15 above)
[the remark this answers is not in the document]
Comment (Shatur) [reply org.kde.CrowTranslate.yaml]: That's what I thought, thanks!
It looks like, in this case, the tessada was installed as part of the main manifest, but the extension relied on a separate, non-existent manifest. The author has just applied the same fix: https://github.com/flathub/io.github.manisandro.gImageReader/pull/71

**Comment by bbhtt** [comment]
The tray menu items are still missing icons

image

**Comment by Shatur** [comment]
Thanks for testing!

I wondering why it happens... I use `window`, `configure` and `application-exit` icons for them respectively. It works in KDE. Are they missing in Advaita?

**Comment by bbhtt** [comment]
The first two doesn't exist.

**Comment by bbhtt** [comment]
You can probably use preferences-other for the configure, it seems to exist in Breeze and Adwaita legacy. The window one doesn't have an option that exists in both other than window-new.

**Comment by Shatur** [comment]
Thanks! Used `preferences-other` for `configure` and `window-maximize` for `window`.

**Comment by Shatur** [comment]
Switched to just using the application tray icon instead of `window-maximize`, I think it fits better.

**Comment by bbhtt** [comment]
bot, build org.kde.CrowTranslate

**Comment by travier** [comment]
bot, build org.kde.CrowTranslate

**Comment by travier** [comment]
Works well for me. LGTM

**Comment by Shatur** [comment]
> After creating a new repository, I will send a PR with a eol rebase manifest to the previous repository. So users will be notified about the appid change.

Should I proceed with this?

**Comment by bbhtt** [comment]
Yea
